modules/persistent-storage-csi-snapshots-operator.adoc

```diff
@@ -5,7 +5,7 @@
 [id="persistent-storage-csi-snapshots-operator_{context}"]
 = About the CSI Snapshot Controller Operator
 
-The CSI Snapshot Controller Operator runs in the `cluster-csi-snapshot-controller-operator` namespace. It is installed by the Cluster Version Operator (CVO) in all clusters by default.
+The CSI Snapshot Controller Operator runs in the `openshift-cluster-storage-operator` namespace. It is installed by the Cluster Version Operator (CVO) in all clusters by default.
```

Comment:
Verified with @jsafrane in 6 April standup that correct namespace is `openshift-cluster-storage-operator` in 4.4+.